queries.sql

```diff
@@ -0,0 +1,19 @@
+
+UPDATE animals SET Neutered = 'false' WHERE Name = 'Pikachu'; -----> this is a request
+SELECT * FROM animals; ---> to check whrther the updated item changed or not
+
+SELECT * FROM animals WHERE Name like '%mon';
+
+SELECT * FROM animals WHERE Date_of_birth >= '2016-02-25' AND Date_of_birth <= '2019-02-27';
```

Comment:
It should list all animals born between 2016 and 2019.
`2016-01-01` to `2019-12-31`